content/about/citing-kokkos.md

```diff
@@ -0,0 +1,105 @@
+---
+authors: ["kokkos-team"]
+title: "Citing Kokkos"
+date: "2025-05-12"
+tags: [""]
+url: "citing-kokkos"
```

Comment:
The URL is unnecessary isn't it?

Reply:
no its necessary so it isn't "kokkos.org/about/citing-kokkos" and it means the URL is not independent of the file location.